### DISCUSSION TITLE
delta-rs forward compatibility with Spark/Flink

### DISCUSSION THREAD

**Discussion by EldarSehayekZenity** [delta-io/delta-rs#2038] [Q&A] [answered]
Hi,

I'm strongly considering adopting delta lake as the OTF for our new data pipeline.
We don't need to go for Spark/Flink yet, but we wish to be forward compatible with them.

Will Spark/Flink be able to handle the delta lake files created by delta-rs?

**Comment by ion-elgreco** [comment, marked as answer]
Spark should be able to read them, we follow the protocol. 

But there are cases where spark-delta doesn't follow the protocol with for example parsing the transaction log to show the history. This is obviously incorrect but has not been addressed yet in spark-delta
